src/WebJobs.Script.WebHost/Controllers/AdminController.cs

```diff
@@ -188,28 +194,31 @@ public override void OnActionExecuting(ActionExecutingContext context)
             base.OnActionExecuting(context);
         }
 
-        // TODO: FACAVAL
-        //[Route("admin/extensions/{name}/{*extra}")]
-        //[HttpGet]
```

Comment:
Leaving this here just to make comparison easier for the PR?

Reply:
This commented out code is what is being replaced by the action implementation.